modules/gapi/perf/cpu/gapi_core_perf_tests_cpu.cpp

```diff
@@ -282,6 +282,53 @@ INSTANTIATE_TEST_CASE_P(ConvertToPerfTestCPU, ConvertToPerfTest,
             Values(0.0),
             Values(cv::compile_args(CORE_CPU))));
 
+INSTANTIATE_TEST_CASE_P(KMeansNDNoInitPerfTestCPU, KMeansNDPerfTest,
+                        Combine(Values(CV_32FC1, CV_32FC3),
+                                Values(cv::Size(1, 20),
+                                       cv::Size(2, 20),
```

Comment:
I think, you can avoid an intermediate value for perf tests.

Reply:
Forgot to delete, thanks - applied
In fact, the parameters has been changed a bit, please take a look